refinery/core/forms.py

```diff
@@ -44,6 +44,22 @@ class RegistrationFormCustomFields(RegistrationForm):
         error_messages={'required': _("You must provide an Affiliation")},
         label=_("Affiliation")
     )
+    spam_filter = forms.CharField(
+        widget=forms.HiddenInput(),
+        required=False
+    )
+
+    def clean_last_name(self):
```

Comment:
Just thinking about this again, and its pretty uncommon for someone to have the same first and last name, but it does happen. Theres Li Li in Paul's lab for example. Maybe the spam filter portion is sufficient?

Reply:
@scottx611x I really think this is fine. Li Li is the first time I've ever encountered the same first and last name ever. But we can mention it in the team meeting for feedback if you're concerned.

Reply:
@jkmarx Alright, its really up to you. Though,  it would probably be easier to allow same first and last names now rather than having to go back and make a change when a user that fits this scenario comes along (as unlikely as it may be)